login-workflow/src/screens/ContactSupport/ContactSupport.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { ReactNode } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import CardActions, { CardActionsProps as CardActionsPropsType } from '@mui/material/CardActions';
+import CardContent, { CardContentProps as CardContentPropsType } from '@mui/material/CardContent';
+import CardHeader, { CardHeaderProps as CardHeaderPropsType } from '@mui/material/CardHeader';
+import Typography, { TypographyProps } from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+import { cx } from '@emotion/css';
+import ChatBubbleOutline from '@mui/icons-material/ChatBubbleOutline';
+import { useLanguageLocale, useInjectedUIContext } from '@brightlayer-ui/react-auth-shared';
+import { BrandedCardContainer } from '../../components';
+import { DialogButtonStyles, DialogActionsStyles, DialogContentStyles, DialogTitleStyles } from '../../styles';
+import { ContactSupportClassKey, getContactSupportUtilityClass, ContactSupportClasses } from './ContactSupportClasses';
+
+const LinkStyles = {
+    fontWeight: 600,
+    color: 'primary.main',
+    textTransform: 'none',
+    textDecoration: 'none',
+    '&:visited': {
+        color: 'primary.main',
+    },
+};
+
+export type ContactSupportProps = {
+    title?: string;
+    TitleProps?: TypographyProps;
+    Actions?: JSX.Element | JSX.Element[];
+    divider?: boolean;
+    ContactSupportContent?: JSX.Element;
+    CardHeaderProps?: CardHeaderPropsType;
+    CardContentProps?: CardContentPropsType;
+    CardActionsProps?: CardActionsPropsType;
+    phoneNumber?: string;
+    emailId?: string;
+    contactSupportQuestion?: string;
+    contactSupportMessage?: string;
+    contactSupportEmergency?: string;
+    contactSupportTechnicalAssistance?: string;
+    ContactSupportIcon?: ReactNode;
+    hideTitle?: boolean;
+    hideContactSupportContent?: boolean;
+    hideContactSupportIcon?: boolean;
+    hideContactSupportQuestion?: boolean;
+    hideContactSupportMessage?: boolean;
+    hideContactSupportEmergency?: boolean;
+    hideContactSupportTechnicalAssistance?: boolean;
+    hideActions?: boolean;
```

Comment:
I'm wondering if all these hidden items can't be put into a single object to simplify the API. Something like this:

```type ContactSupportHiddenElements = {
    title?: boolean;
    contactSupportContent?: boolean;
    contactSupportIcon?: boolean;
    contactSupportQuestion?: boolean;
    contactSupportMessage?: boolean;
    contactSupportEmergency?: boolean;
    contactSupportTechnicalAssistance?: boolean;
    actions?: boolean;
};```

```type ContactSupportProps = {
    // ...
    hiddenElements?: ContactSupportHiddenElements;
    // ...
};```

```const {
        // ...
        hiddenElements = {},
        // ...
    } = props;```

and then all your conditionals would update like this:
 ```{!hiddenElements.title && (
                // ...
            )}
            {!hiddenElements.contactSupportContent && (
                // ...
            )}```